DEVELOPER_GUIDE.rst

```diff
@@ -405,7 +405,7 @@ Sample test class:
 Doctest
 >>>>>>>
 
-Python doctest library makes our document executable which keeps it up-to-date to source code. The doc generator aforementioned served as scaffolding and generated many docs in short time. Now the examples inside is changed to doctest gradually. For more details please read `Doctest <./dev/Doctest.md>`_.
+Python doctest library makes our document executable which keeps it up-to-date to source code. The doc generator aforementioned served as scaffolding and generated many docs in short time. Now the examples inside is changed to doctest gradually. For more details please read `testing-doctest <./docs/dev/testing-doctest.md>`_.
```

Comment:
@currantw just making sure - why was changed ?

Reply:
I noticed that this was incorrect when reading through this documentation. There is no longer a `Doctest.md` file in the code base, so I have updated this to point to the correct file.

Reply:
Noticed while reading through this that `Doctest.md` no longer exists, so corrected the link.